SignifyService/DetectEngine/signify.py

```diff
@@ -35,7 +39,7 @@
 # ====== Neural Network
 str_device = "cuda" if torch.cuda.is_available() else "cpu"
 device = torch.device(str_device)
-model = torch.hub.load('ultralytics/yolov5', 'custom', path='../Weights/best.pt')
+model = torch.hub.load('ultralytics/yolov5', 'custom', path=(abs_dir + '/../Weights/best.pt'))
```

Comment:
Seems like you handled changes to the 'abs_dir' manually, but you also changed the file locations, therefore your manual changes may fail. (for example, if signify.py were to change where it was, this line would be ok. but because you also changed this file's location, Im not sure it will still be looking for the weights in the right place.